cumulusci/tasks/apex/testrunner.py

```diff
@@ -222,15 +230,36 @@ def _get_test_classes(self):
         self.logger.info("Found {} test classes".format(result["totalSize"]))
         return result
 
-    def _get_test_results(self):
+    def _is_retriable_failure(self, test_result):
```

Comment:
https://english.stackexchange.com/questions/305273/retriable-or-retryable convinced me to let this stand :)